components/ui/sidebar.tsx

```diff
@@ -20,11 +22,11 @@ const Sidebar = () => {
           <LogoDark className='mb-5' />
         </div>
         {/* Navigation items */}
-        <SidebarNavItem title='Dashboard' firstItem isActive />
-        <SidebarNavItem title='Wallet'  />
-        <SidebarNavItem title='Bridge' />
-        <SidebarNavItem title='Staking' isActive  />
-        <SidebarNavItem title='Governance' />
+        <SidebarNavItem title='Dashboard' firstItem />
+        <SidebarNavItem title='Wallet' href='wallet' />
```

Comment:
Why not prefixing the `href` attribute with a `/` for internal navigation paths? This will make the value consistent with the default root path ("/") and simplifiy the comparison logic for determining the active state of the `SidebarNavItem` component.

Reply:
done